WordPress/WordPressTest/ReaderTopicServiceTest.swift

```diff
@@ -138,6 +157,51 @@ final class ReaderTopicSwiftTest: XCTestCase {
 
     // MARK: Tests
 
+    /**
+    Ensure that followed sites a user unfollows from are set to unfollowed in core data when merging
+    results from the REST API.
+    */
+    func testUnfollowedSiteIsUnfollowedDuringSync() {
+        guard let context = context else {
```

Comment:
Wdyt about separating this test in 3 parts using comments? An example: https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/WordPressTest/PostCoordinatorTests.swift#L369

You can use Arrange/Act/Assert or Given/Then/When, whichever you like the most.

I find them useful in big tests because I can clearly see the 3 different sections. Here, for example, it took me a while to understand that you were first populating the sites and then testing. :)

Reply:
Thanks for the suggestion! I totally agree, this is kind of a big test so I'll divide it up into 3 sections 😄

Reply:
Fixed! 6d4135c